src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -0,0 +1,566 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for MaskFormer."""
+
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+from PIL import Image
+
+from ...feature_extraction_utils import BatchFeature, FeatureExtractionMixin
+from ...file_utils import TensorType, is_torch_available
+from ...image_utils import ImageFeatureExtractionMixin, ImageInput, is_torch_tensor
+from ...utils import logging
+
+
+if is_torch_available():
+    import torch
+    from torch import Tensor, nn
+    from torch.nn.functional import interpolate
+
+    from transformers.models.maskformer.modeling_maskformer import MaskFormerForInstanceSegmentationOutput
```

Comment:
This is a circular import waiting to happen, and is just used for type hints, so this import should be removed (and type hints use strings).

Reply:
Due to types, even if I change it to string, it needs to be there otherwise `make quality` fails

Reply:
No, it only needs to be imported under a `if TYPE_CHECKING` block which will avoid a circular import.

Reply:
understood, thanks for the clarification